modular_skyrat/modules/customization/modules/surgery/organs/taur_body.dm

```diff
@@ -88,6 +88,14 @@
 
 	can_ride_saddled_taurs = TRUE
 
+/obj/item/organ/external/taur_body/anthroRideable
```

Comment:
firstly, wrong formatting, we use snake_case not camelCase

secondly, as i said, you do not need to make a new subtype. just modify taur_body/anthro to have these vars

Reply:
ah sorry. The issue with just modifying the existing taur type is that if you are rideable yourself you cannot ride other taurs. So i made a duplicate set that are rideable

Reply:
can_ride_saddled_taurs = TRUE

Reply:
Yeah i don't want any clown car shenanigans going on  unless im missing something.

Reply:
Didnt you jsut say you want this subtype to be rideable and to be able to ride? I dont get waht you're saying. But regardless, a subtype isnt needed, if you want it to be rideable but not be able to ride, you can set can_ride_saddled_taurs to FALSE. Which is the default.

Reply:
Sorry I just slept but I will be clear now.
I want a type that can ride others and another type that is rideable but can not ride others. 
The reason for the alternative type is to set pixel shifting for riders to be situated closer to the taur's body due to a lack of horselike body.